scalafmt-core/jvm/src/main/scala/org/scalafmt/internal/PriorityQueue.scala

```diff
@@ -0,0 +1,37 @@
+package org.scalafmt.internal
+
+import java.util.Comparator
+
+/**
+  * Minimal implementation of the PriorityQueue's functions needed.
+  *
+  * We use [[java.util.PriorityQueue]] to enable usage under GraalVM. The
+  * native-image compiler is unable to work with
+  * [[scala.collection.mutable.PriorityQueue]] currently.
+  *
+  * @tparam T the values inside the queue
+  */
+class PriorityQueue[T <: Ordered[T]] {
+  private[this] val q =
+    new java.util.PriorityQueue[T](11, new Comparator[T] {
+      override def compare(t: T, t1: T): Int = t1.compare(t)
```

Comment:
Attention: here I'm doing `t1.compare(t)`, but it seems that this code passes tests (locally and in Travis), while `t.compare(t1)` generates errors. I'm thinking this is because the PriorityQueue in Scala wants elements in the reverse order than Java's.